lib/capistrano/tasks/assets.rake

```diff
@@ -62,15 +62,29 @@ namespace :deploy do
       end
     end
 
+    def manifest_path(filename)
+      release_path.join('public', fetch(:assets_prefix), filename)
+    end
+
+    def with_backup_and_manifest_paths &block
+      backup_path = release_path.join('assets_manifest_backup')
+      manifest_path = test(:ls, manifest_path('.sprockets-manifest*'))
+      if manifest_path
+        manifest_path = capture(:ls, manifest_path('.sprockets-manifest*')).strip
+      else
+        manifest_path = capture(:ls, manifest_path('manifest*')).strip
+      end
+
+      yield manifest_path, backup_path
```

Comment:
I don't think we need it--matter of personal taste really.  I think it looks cleaner but will happily change it if you feel strongly about it.